src/components/article.tsx

```diff
@@ -16,13 +17,22 @@ type Props = {
 const Article = ({
   title,
   html,
+  tableOfContents,
   previous,
   next,
   relativePath,
   authors,
 }: Props) => (
   <article className="article-reader">
     <h1 className="article-reader__headline">{title}</h1>
+    {tableOfContents && (
```

Comment:
Just a thought here, we could split this out to a separate TOC-component 🙂

Reply:
Done in b1d1719